src/Interfaces/DynamoTestStation.ts

```diff
@@ -0,0 +1,17 @@
+interface DynamoTestStation {
+  testStationId : string,
```

Comment:
```suggestion
  testStationId: string,
```

Reply:
Thanks, need to turn my linter back on

Reply:
I was thinking that the linter would have spotted this.

Reply:
I disabled the plugin to look at the other project